### ISSUE TITLE
This project will shuffle the data list or not during training?

### ISSUE THREAD

**Issue by gaopeng-eugene** [DrSleep/tensorflow-deeplab-resnet#42]
It seems like there is shuffle in training while no shuffle in testing.

self.queue = tf.train.slice_input_producer([self.images, self.labels],
                                                   shuffle=input_size is not None) # not shuffling if it is val

However, when I train the model several times the loss is totally same which means there is no shuffle in training.

**Comment by DrSleep** [comment]
https://github.com/DrSleep/tensorflow-deeplab-resnet/commit/a877f5d6e2cdecc26c625b53676f78f87673c429

**Comment by gaopeng-eugene** [comment]
I am not quite clear about the RANDOM_SEED. Do you mean we need to change the number when training?

**Comment by mokii** [comment]
I wonder about it too.Do you mean we need to change the number when training?Will changing it brings more performance gain?
